src/components/NewFileDialog.tsx

```diff
@@ -78,7 +78,7 @@ export class NewFileDialog extends React.Component<{
     return <ReactModal
       isOpen={this.props.isOpen}
       contentLabel="Create New File"
-      className="modal"
+      className="modal show-file-icons"
```

Comment:
This is necessary because modals are outer elements, not wrapped by the container with "show-file-icons".